department-of-veterans-affairs/vets-website
# Bah z caftercheck

## Description
1. On the (Search Results page)[https://staging.va.gov/gi-bill-comparison-tool/profile/11801720], user selects "Other location" in response to "Where will you take the majority of your class", "Please enter the Postal code where you'll take your classes" appears.
2. When user enters an invalid zip (anything other than 5 numbers) and navigates away from the field, the following error message displays: "Zip code must be a 5-digit number"

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19674

## Testing done
1. Navigate to http://localhost:3001/gi-bill-comparison-tool/profile/11806124
activate voice over
2. Find the question Where will you take the majority of your classes?
3. Select the "An extension campus" radio button
4. Select other from the "Choose the location where you'll take your classes" drop down.
5. Enter less than 5 digits in the Zip code field and tab off
6. Error message "Zip code must be a 5-digit number" will appear

## Screenshots
<img width="492" alt="Screen Shot 2019-09-12 at 9 58 25 AM" src="https://user-images.githubusercontent.com/50601724/64790371-fd420c00-d543-11e9-8dcc-d942454222ca.png">



## Acceptance criteria
- [x] When user clicks away from the zip code field with an invalid zip code entered, the following error message is displayed: "ZIP Code must be a five digit number"

## Definition of done
